### PR TITLE
fix: Update Ticket and Registration Section on Event Public Page

### LINKED ISSUE #7809: Registrations: Move tax info below price
The tax is currently not below the actual price. Please move it there. Show it similarly to how it is displayed for ticket buyers (here we are looking at registrations).

Screenshot from 2021-09-19 20-35-47
Screenshot from 2021-09-19 19-52-04

### LINKED ISSUE #7814: Change text of Subtotal, included and Grand Total
On the ticket section and on the orders page place change the text as shown in the red ink in the below screenshot.

Screenshot from 2021-09-19 22-16-01

### LINKED ISSUE #7815: Ticket Section: Donation ticket does not show tax
When entering an amount for a donation ticket or registration the tax (included, or on top) is not updated. Expected: When the user chooses the number of donation tickets or the registration update the tax amount.

Screenshot from 2021-09-19 22-40-53

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Fixes #7815 
Fixes #7814
Fixes #7809 

#### Short description of what this resolves:

- [x] Update Tax amount showing below text box on updating ticket price.
- [x] Change text of Subtotal, included and Grand Total
- [x] Registrations: Move tax info below price

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by daretobedifferent18** [comment]
@mariobehling this issue is only for when tax is included and working fine for tax on top.
Fixed now.

**Comment by daretobedifferent18** [comment]
Desktop | Mobile
-------- | -------
Screenshot from 2021-09-20 06-05-28 | Screenshot from 2021-09-20 06-06-16

**Comment by daretobedifferent18** [comment]
Screenshot from 2021-09-20 06-12-24

File: app/models/ticket.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default ModelBase.extend({
    * when tax amount is included in ticket price, otherwise return
    * 0
    */
-  includedTaxAmount: computed('event.tax.isTaxIncludedInPrice', 'event.tax.rate', function() {
+  includedTaxAmount: computed('price', 'event.tax.isTaxIncludedInPrice', 'event.tax.rate', function() {
```
Comment (maze-runnar) [inline app/models/ticket.js:68]: why include price here?

Comment (daretobedifferent18) [reply app/models/ticket.js:68]: bcz on changing the price of donation ticket, we need to update the tax amount

**Comment by maze-runnar** [comment]
@daretobedifferent18 resolve conflict

**Review by maze-runnar — APPROVED** [review]

**Pull request merged**